semgrep/Pipfile

```diff
@@ -11,6 +11,7 @@ pytest-snapshot = "*"
 
 [packages]
 semgrep = {editable = true,path = "."}
+attrs = "*"
```

Comment:
You probably want to add this to setup.py

Reply:
oh good call.